OpenTabletDriver.Configurations/Configurations/XP-Pen/Artist 12 (3rd Gen).json

```
@@ -0,0 +1,31 @@
+{
+  "Name": "XP-Pen Artist 12 (3rd Gen)",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 264.00,
+      "Height": 149.00,
+      "MaxX": 52638,
+      "MaxY": 29616
+    },
+    "Pen": {
+      "MaxPressure": 16384,
```

```suggestion
      "MaxPressure": 16383,
```
this is usually 1 below the pressure range stated by the manufacturer, as `0` is also a valid pressure number

`grep` shows:
```
$ grep -R 16383
Gaomon/M8 (Variant 2).json:      "MaxPressure": 16383,
Huion/HC16 (Variant 2).json:      "MaxPressure": 16383,
Huion/Kamvas Pro 27.json:      "MaxPressure": 16383,
Huion/L310.json:      "MaxPressure": 16383,
Huion/Kamvas 13 (Gen 3).json:      "MaxPressure": 16383,
Huion/Kamvas 16 (Gen 3).json:      "MaxPressure": 16383,
Huion/Kamvas Pro 19 (4K).json:      "MaxPressure": 16383,
Huion/L610.json:      "MaxPressure": 16383,
XP-Pen/Deco Pro LW Gen2.json:      "MaxPressure": 16383,
XP-Pen/Deco Pro XLW Gen2.json:      "MaxPressure": 16383,
XP-Pen/Deco 640 (IT640).json:      "MaxPressure": 16383,
XP-Pen/Deco Pro MW Gen2.json:      "MaxPressure": 16383,
<snip>
```